redash/schedule.py

```diff
@@ -11,26 +15,36 @@
                           cleanup_query_results,
                           version_check, send_aggregated_errors)
 
+logger = logging.getLogger(__name__)
+
 rq_scheduler = Scheduler(connection=rq_redis_connection,
                          queue_name="periodic",
                          interval=5)
 
+def job_id(kwargs):
+    metadata = kwargs.copy()
+    if 'func' in metadata:
+        metadata['func'] = metadata['func'].__name__
```

Comment:
Can there be a job definition without `func`? Also, isn't the func name good enough as a job id?

Reply:
Not sure what I was thinking with checking for existence in metadata.

if we use the function name as an identifier, then we would have to change the function name when we want to apply other things (for example, the interval or results TTL) in order to reschedule them.

Reply:
How about the job id becomes `metadata['func'].__name__:hash`? This way when poking around and seeing a job id it's easy to tell what job it references?